kornia/enhance/adjust.py

```diff
@@ -600,6 +600,41 @@ def adjust_log(image: Tensor, gain: float = 1, inv: bool = False, clip_output: b
     return img_adjust
 
 
+def auto_contrast(image: Tensor, clip_output: bool = True) -> Tensor:
+    r"""Apply auto-contrast.
+
+    The image is expected to be an RGB image or a gray image in the range of [0, 1].
+
+    Args:
+        image: Image/Tensor to be adjusted in the shape of :math:`(*, 1, H, W)` or :math:`(*, 3, H, W)`.
+        clip_output: Whether to clip the output image with range of [0, 1].
+
+    Return:
+        Adjusted image in the shape of :math:`(*, 1, H, W)` or :math:`(*, 3, H, W)`.
+    
+    ..note:
+        Currently NON-differentiable.
+    """
+    KORNIA_CHECK_IS_TENSOR(image, "Expected shape (*, H, W)")
+    KORNIA_CHECK_IS_COLOR_OR_GRAY(image, "Image should be an RGB or gray image")
+
+    # Taken from pytorch transforms/functional_tensor.py
+    with torch.no_grad():
```

Comment:
why `kornia.enhance.normalize_min_max` doesn't work ?

Reply:
I compared `kornia.enhance.normalize_min_max` and `torchvision.transforms.functional.autocontrast` on a few examples and they gave same results. Did not compare to PIL's autocontrast.

Reply:
@miquelmarti. Great. We will use our version then.